src/app/models/course.rb

```diff
@@ -0,0 +1,3 @@
+class Course < ApplicationRecord
+  belongs_to :user
```

Comment:
I think this user is the supervisor. In addition the students are needed. I think it is therefore necessary to rename the attributes because there are then 2 relations to the user table